### PR TITLE
Feature/aturuk

### PR DESCRIPTION
**Pull request by aTurukDCCA**
Added small file utility

### CODE REVIEW

**Review by farruhOsman — COMMENTED** [review]
target folder should be in gitignore.

**Review by farruhOsman — APPROVED** [review]

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,3 @@
+
+.idea
+*/target
```
Comment (rachowdhury-dcca) [inline .gitignore]: We want to add *.class files right to get rid of the java binaries.

File: POC_API/src/test/java/org/Execute.java [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+import org.utilities.FileManager;
+
+import java.util.List;
+import java.util.Map;
+
+public class Execute {
+    public static void main(String[] args) {
+        FileManager.createFileCopy("C:\\Users\\aturuk\\IdeaProjects\\mtf-automation-testing-api\\confirmClone.txt",
+                "C:\\Users\\aturuk\\IdeaProjects\\mtf-automation-testing-api\\confirmCloneDup.txt");
+
```
Comment (rachowdhury-dcca) [inline POC_API/src/test/java/org/Execute.java:12]: You should be able to use relative paths so that other's can run this (they have different usernames).

Comment (aTurukDCCA) [reply POC_API/src/test/java/org/Execute.java:12]: this is just sample class to test/debug  the  functions

Comment (aTurukDCCA) [reply POC_API/src/test/java/org/Execute.java:12]: we don't really need it..

**Review by rachowdhury-dcca — APPROVED** [review]
Since we don't have coding standards yet and I don't want to block what you're trying to do but later when we work in the CMS github, we should be more strict I think.

**Comment by aTurukDCCA** [comment]
> Since we don't have coding standards yet and I don't want to block what you're trying to do but later when we work in the CMS github, we should be more strict I think.

%100

File: .gitignore (hunk 1 above)
Comment (aTurukDCCA) [reply .gitignore]: added

**Review by yasamanibrahimova — APPROVED** [review]

**Review by yasamanibrahimova — APPROVED** [review]

**Review by TamerlanHuseynov-DCCA — APPROVED** [review]

**Review by yasamanibrahimova — APPROVED** [review]

**Review by yasamanibrahimova — APPROVED** [review]

**Review by yasamanibrahimova — APPROVED** [review]

**Pull request closed without merging**